### ISSUE TITLE
Compile data - URCK catch

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [pfmc-assessments/canary_2023#27]
Moved from #9 to its own issue

**Comment by brianlangseth-NOAA** [comment]
@KSten and Theresa

Per our email exchange in November I want to keep the same approach for WA landings from PacFIN as was done in the last assessment. I was able to dig a little bit with the help of @chantelwetzel-noaa, who pulled the URCK pacfin compFT file, and provide the details below for reference. It does not change my intent of keeping the approach the same. As we discussed via email, having something consistently applied across all species would be much more comprehensive than individual choices. 

For WA, nearly all landings from 1981-1987 that are assigned as canary come from URCK, (though during the 1988-1992 a lot come from ORCK) 

When looking at the remnant URCK catches that chantel pulled, the remaining total amounts range from 180%-30% of the catches that were assigned to canary during 1981-1993, with a declining trend over time. Consequently, the most meaningful timeframe for possibly allocating any remaining unidentified catch for canary is 1981-1987. 

I did not look at whether there is bds data for URCK, in part given my desire to keep the same approach as for 2015.
